src/bundle/Resources/views/themes/admin/ui/search/form.html.twig

```diff
@@ -19,6 +22,74 @@
             <span class="ez-btn ez-btn--filter-label">{{ 'search.filters'|trans|desc('Filters') }}</span></button>
     </span>
 </div>
+
+<div class="ez-search-criteria-tags ml-4 {% if filters_expanded %} ez-search-criteria-tags--collapsed{% endif %}">
+    {% if form.content_types.vars.data is not empty %}
+        {% for contentType in form.content_types.vars.data %}
+            {{ include('@ezdesign/ui/search_tag.html.twig', {
+                'content': contentType.name,
+                'title': "#{'search.content.type'|trans|desc('Content Type:')} #{contentType.name}",
+                'reletedFiledId': "search_content_types_#{contentType.identifier}",
```

Comment:
Maybe we could do `targetSelector/fieldSelector` and pass the whole selector not only the id?